community/samples/README.md

```diff
@@ -20,8 +20,7 @@ Knative Serving sample apps.
 
 #### Eventing and Eventing Resources samples
 
-- _Be the first to contribute an Eventing or Eventing Sources code sample to the
-  community collection._
+| Hello World Eventing | A quick introduction to Knative Eventing with HTTP and Kafka Source binding using Cloud Events | [Blog Post](https://salaboy.com/2020/02/20/getting-started-with-knative-2020/), [Event Producer using Spring Cloud Streams & Kafka](https://github.com/salaboy/knative-spring-cloud-event-producer) and [HTTP Cloud Event producer using Spring Boot](https://github.com/salaboy/knative-event-producer) |
```

Comment:
the formatting is a bit odd ?

Reply:
@matzew I've copied the table from the previous section. It has the same formatting. Can you suggest me a better formatting for that?

Reply:
it is not just odd, it does not render properly as far as I can see: https://github.com/knative/docs/blob/fb6bbe4c00c9f76f63b4438d51a686beb4205b90/community/samples/README.md

Reply:
first the section heading should only be 3 # , then you are missing top level categories underlined b y dashes.

Reply:
@sebgoa thanks for letting me know... let me fix that issue.